cmd/bisync/cmd.go

```diff
@@ -50,6 +50,12 @@ type Options struct {
 	Resilient             bool
 	TestFn                TestFunc // test-only option, for mocking errors
 	Retries               int
+	GUI                   bool
```

Comment:
Being able to show status to the user is a great idea.

However I'd like to consider the other operations too like sync etc.

At the moment you can build a pretty good gui with javascript using the rc.

So I think what you should do is take this commit off the series and stash it on a branch, then make an issue for discussion.

Reply:
Ok, I'll remove it for now and make an issue for discussion. I do think there are several bisync-specific things that wouldn't make sense sharing a template with `sync` (for instance, the Path1 / Path2 columns with links to their respective listing files.) ([Demo](https://github.com/nielash/rclone/blob/bisync-logger/docs/static/img/bisync_gui_example.png)) It also displays a warning if a bisync was locked out by a prior run -- which is not a thing in sync.

The rc is very nice but I think it's a drawback that the server needs to be running in order to use it, vs. a static HTML file which can live anywhere (even another rclone remote). I also think the rc adds complexity for new users who want to get up and running quickly. But I'll move it to a branch for now for further discussion!